spec/formatted_text_formatter_spec.rb

```diff
@@ -210,8 +212,8 @@
       (expect output[0][:link]).to eql 'https://cast.you?v=999999&list=abcde&index=1'
     end
 
-    it 'should decode hexidecimal character references in link href' do
-      output = subject.format '<a href="https://cast.you?v=999999&#x26;list=abcde&#x26;index=1">My Playlist</a>'
+    it 'should decode hexadecimal character references in link href' do
+      output = subject.format '<a href="https://cast.you?v=999999&#x26;list&#x3d;abcde&#x26;index&#x3d;1">My Playlist</a>'
```

Comment:
I don't think this change is needed. The test is already asserting that the hexadecimal character references are decoded.